src/test/java/io/specto/hoverfly/junit/core/HoverflyTest.java

```diff
@@ -32,8 +35,12 @@
     private Hoverfly hoverfly;
     private ObjectMapper mapper = new ObjectMapper();
 
+    @Rule
+    public final RestoreSystemProperties restoreSystemProperties = new RestoreSystemProperties();
```

Comment:
No I haven't. Feel free to work on this issue: https://github.com/SpectoLabs/hoverfly-java/issues/70
Thanks!